recipes/ipumspy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ipumspy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e64249ba3e22a84a99ded0d630dc5977c7b5d464a4074a8407c39c3f2ccc4011
+
+build:
+  number: 0
+  entry_points:
+    - ipums = ipumspy.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```

Comment:
You don't need this skip b/c we don't build py2k. Also, you probably can make this one noarch. You'll have to drop the py38 selectors below.

Reply:
Thanks @ocefpaf , I have addressed these.